### PR TITLE
FE build pipeline

### PR DESCRIPTION
**Pull request by clloyd**
@philmcmahon 

This adds a `npm run start` and `npm run build` command.

TODO:
- [x] Readme

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,3271 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (ShaunYearStrong) [inline yarn.lock]: Doesn't this file need ignoring?

File: setup.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    echo -e "Yarn is not required for the application (but is for this script)"
+    echo -e "Packages can be manually installed with npm\n\r\n\r"
+
+    exit 1
+fi
+
+printf "\n\rSetting up client side dependancies... \n\r\n\r"
+printf "\n\rInstalling NPM packages via yarn... \n\r\n\r"
+
+yarn
```
Comment (ShaunYearStrong) [inline setup.sh]: 🎉

**Comment by ShaunYearStrong** [comment]
👍

File: yarn.lock (hunk 1 above)
Comment (clloyd) [inline yarn.lock]: Nope! One of the advantages of yarn is the lock file ensures we all use the exact same dependencies, designed to be checked in https://yarnpkg.com/en/docs/yarn-lock

**Pull request merged**